procedures/katello/release.md.erb

```diff
@@ -4,21 +4,30 @@
 
 - [ ] Request Hammer CLI Katello release from maintainers
 - [ ] Request Virt Who Configure release from maintainers
-<% unless is_rc %>- [ ] Change Redmine version <%= full_version %> state to Closed
+- [ ] Request any other releases as needed from the release owners (consult the `:repos:` section of `configs/katello/<%= short_version %>.yaml`)
+<% unless is_rc %>
+- [ ] Change Redmine version <%= full_version %> state to Closed
 <% end %>
-- [ ] Clone [tool_belt](https://github.com/theforeman/tool_belt) and run:
-  - [ ] `./tools.rb setup-environment configs/katello/<%= short_version %>.yaml`
-  - [ ] `./tools.rb cherry-picks --version <%= full_version %> configs/katello/<%= short_version %>.yaml`
-  - [ ] Verify tickets in the cherry_picks_<%= full_version %> file are accounted for or additional cherry pick them
-  - [ ] Check for outdated deprecation warnings in the current and next release with `./tools check-deprecation-warnings configs/katello/<%= short_version %>.yaml`. Follow the instructions in the output of the command.
-
-- [ ] In katello release branch:
-  - [ ] Update `lib/katello/version.rb` to <%= full_version %>
+- [ ] Do Cherry-picks: Clone [tool_belt](https://github.com/theforeman/tool_belt)
+  - [ ] If any minor versions of Katello have been added to Redmine since the last cherry-pick, make sure to include them in `prior_releases` in configs/katello/<%= short_version %>.yaml
+  - [ ] Run `./tools.rb setup-environment configs/katello/<%= short_version %>.yaml`
+  - [ ] Run `./tools.rb cherry-picks --version <%= full_version %> configs/katello/<%= short_version %>.yaml`
+  - [ ] Open a PR in Katello release branch.  Make sure the PR name starts with [CP] to prevent our automations from adding it to Redmine issues.
+  - [ ] Using `git cherry-pick -x` as needed, verify tickets in the cherry_picks_<%= full_version %> file are accounted for, or additionally cherry-pick them.  Recommended: Do this in tool_belt's checkout of Katello, in `repos/katello/<%= full_version %>/katello`. This way when you run cherry-picks again, tool_belt will be aware of any picks already completed.
+  - [ ] For any cherry-picks that are not needed (including Redmine trackers) you can add them to the `:ignores:` section of `tool_belt` in `configs/katello/<%= short_version %>.yaml`
+- [ ] Check for outdated deprecation warnings in the current and next release with `./tools check-deprecation-warnings configs/katello/<%= short_version %>.yaml`. Follow the instructions in the output of the command. Don't forget to create any Redmine issues needed!
+
+- [ ] Bump version:
+  - [ ] Open a PR (or use cherry-pick PR) against the release branch which updates `lib/katello/version.rb` to <%= full_version %>
```

Comment:
Why is a PR needed for this? We never do this for Foreman and just straight commit it.

Reply:
I just don't like committing directly to the upstream unless I have to. @jturel thoughts?

Reply:
The release owner might not have permission to push directly to branches.

Reply:
My goal is to align both Foreman and Katello release processes as much as possible so I do think we should follow the same steps.

Eventually I hope we can all standardize on calling [a script](https://github.com/theforeman/theforeman-rel-eng/blob/master/tag_project).